data/model/data-query.js

```diff
@@ -96,6 +96,15 @@ exports.DataQuery = Montage.specialize(/** @lends DataQuery.prototype */ {
         value: undefined
     },
 
+    /**
+     * A property used to give a meaningful name to an operation
+     *
+     * @type {string}
+     */
+    name: {
```

Comment:
Per the comment, is this meant to be on DataOperation instead of DataQuery? 

Or if it's meant to be DataQuery, can you give an example of it's usage?